modules/lib/src/main/resources/assets/js/app/wizard/page/LiveFormPanel.ts

```diff
@@ -635,7 +634,7 @@ export class LiveFormPanel
         }
     }
 
-    saveAndReloadOnlyComponent(componentView: ComponentView<Component>) {
+    saveAndReloadOnlyComponent(componentView: ComponentView<Component>, avoidInspectComponentRefresh?: boolean) {
```

Comment:
For boolean properties it's better to use default param value rather than sending in `undefined`.

`saveAndReloadOnlyComponent(componentView: ComponentView<Component>, avoidInspectComponentRefresh: boolean = false)`